src/app/core/basics/notification.service.ts

```diff
@@ -1,17 +1,18 @@
 import {Injectable} from '@angular/core';
 import {BehaviorSubject, combineLatest, of} from 'rxjs';
-import {VbsSubmissionService} from '../vbs/vbs-submission.service';
+import {VbsSubmissionService} from '../competition/vbs-submission.service';
```

Comment:
I very much welcome the refactoring and thus renaming of the module. However, I wonder why the actual submitting service was not renamed?

Reply:
Because at some point it should be significantly refactored and that would require significantly more work than the scope of the PR